lib/expose.cjs

```diff
@@ -0,0 +1,17 @@
+// https://stackoverflow.com/a/46745166/10109857
+
+/**
+ * returns renovates package.json
+ * @type {import('./types').RenovatPackageJson}
+ */
+const pkg = (() => require('../package.json'))();
+
+/**
+ * return's re2
+ * @returns {RegExpConstructor}
+ */
+function re2() {
+  return require('re2');
```

Comment:
We have over 800 references to `regEx`, so i don't like to make it async. so this is a esm workaround, as this file is always interpreted as cjs. 🙃